code/game/objects/items/devices/cictablet.dm

```diff
@@ -17,6 +17,7 @@
 	var/announcement_title = COMMAND_ANNOUNCE
 	var/announcement_faction = FACTION_MARINE
 	var/add_pmcs = TRUE
+	var/cooldown_request = 0
```

Comment:
use the COOLDOWN_DECLARE system (much simpler) (it's already used here with the announcement_cooldown)